nestjs/src/courses/course-schedule/course-schedule.service.ts

```diff
@@ -70,6 +70,33 @@ export class CourseScheduleService {
     readonly taskCheckerRepository: Repository<TaskChecker>,
   ) {}
 
+  private scheduleSort(a: CourseScheduleItem, b: CourseScheduleItem) {
+    const tagPriority: { [key in CourseScheduleItemTag]?: number } = {
+      'self-study': 1,
+      test: 2,
+      coding: 3,
+    };
+
+    // Function to get the time in minutes (ignoring seconds and milliseconds)
+    const getTimeInMinutes = (date: Date) => {
+      return new Date(
+        date.getFullYear(),
+        date.getMonth(),
+        date.getDate(),
+        date.getHours(),
+        date.getMinutes(),
+      ).getTime();
+    };
+
+    const timeDifference = getTimeInMinutes(a.startDate) - getTimeInMinutes(b.startDate);
```

Comment:
is it somehow different from `dayjs.diff()`? 
```js 
date1.diff(date2, 'minute')
```

Reply:
Thank you, mate. I forgot about dayjs))